Source/Meadow.Core.Samples/Board_Specific_Samples/F7_Micro/F7_Micro_Board_Diagnostics/MeadowApp.cs

```diff
@@ -14,12 +13,12 @@ public override Task Initialize()
         {
             // Simple Digital IO
             Tuple<bool, List<PortTestResult>> digitalIOResults = TestDigitalIO();
-            Debug.WriteLine("Simple Digital IO Test Results: " + (digitalIOResults.Item1 ? "PASS" : "FAIL"));
+            Console.WriteLine("Simple Digital IO Test Results: " + (digitalIOResults.Item1 ? "PASS" : "FAIL"));
```

Comment:
Do we want to go the next step to `Resolver.Log.{whatever}`?

Reply:
I can, replacing 210 occurrences :)

Reply:
done